packages/gr/gr.0.0.2/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings to the GR plotting library"
+description: """
+GR is based on an implementation of a Graphical Kernel System (GKS) and OpenGL. As a self-contained system it can quickly and easily be integrated into existing applications.
+GR is characterized by its high interoperability and can be used with modern web technologies and mobile devices."""
+maintainer: ["Marcello Seri"]
+authors: ["Marcello Seri"]
+license: "ISC"
+homepage: "https://github.com/mseri/ocaml-gr"
+bug-reports: "https://github.com/mseri/ocaml-gr/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "1.11"}
+  "ctypes" {>= "0.15"}
+  "ctypes-foreign" {true}
+  "base-unix" {with-test}
+  "owl" {with-test & >= "0.6.0"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```
These are just compiling the examples to check that they are in sync with the API